server/src/db/migrations/20220119232016_store-front.ts

```diff
@@ -0,0 +1,123 @@
+import { Knex } from "knex";
+
+
+export async function up(knex: Knex): Promise<void> {
+
+    knex.schema.hasTable('InventoryItem').then(function (exists) {
```

Comment:
This is just a stylistic question, but is there a reason we're doing `function (exists) {` instead of `(exists) => {`?

Reply:
I probably just copied this from Knex documentation. Arrow functions look a bit cleaner so I might update